app/models/merchant.rb

```diff
@@ -1,6 +1,8 @@
 class Merchant <ApplicationRecord
   has_many :items, dependent: :destroy
   has_many :item_orders, through: :items
+  has_many :orders, through: :item_orders
+  has_many :users, -> { where(role: 1) }
```

Comment:
Ash you knew what you were doing! Shouldn't have doubted you. has many relation was the way to go